__tests__/components/ApplicationCommentsSummary.test.tsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { ApplicationCommentsSummary } from "@/components/ApplicationCommentsSummary";
+import {
+  PublicCommentSummary,
+  SpecialistCommentSummary,
+} from "@/types/odp-types/schemas/postSubmissionApplication/data/CommentSummary";
+
+jest.mock("@/components/SentimentIcon", () => ({
```

Comment:
Your missing mocks for `Button`, `capitalizeFirstLetter`, `pascalToSentenceCase` and `Link` in case anything is wrong in those files!